.swm/doc-templates/component-or-service-overview-pqkt4.sw.md

```diff
@@ -0,0 +1,28 @@
+---
+id: pqkt4
+name: Component or Service Overview
+file_version: 1.0.2
+app_version: 0.7.1-1
+---
+
+## Introduction
+This doc gives a high level overview of {{COMPONENT NAME}}. It is located under {{use `/path` to select the folder where the component / service is implemented}}.
+
+## Main features
+The main features of {{COMPONENT NAME}} are:
+
+## Interface
+{{How can this component/service be accessed?}}
+
+<br/>
+<!-- TEMPLATE-swimm-snippet-placeholder -->
+Select an example of accessing this component/service
```

Comment:
Instructions. should be {{Select an example of accessing this component/service}}

Reply:
@nadav-swimm this is within the snippet placeholder - so it appears within the placeholder.. I think we shouldn't use {{ for these...